app/helpers/concerns/foreman_bootdisk/hosts_helper_ext.rb

```diff
@@ -69,55 +64,30 @@ def host_action_buttons(host)
 
       host_image_link = display_bootdisk_link_if_authorized(
                           _("Host '%s' image") % host.name.split('.')[0],
-                          {
-                              controller: 'foreman_bootdisk/disks',
-                              action: 'host',
-                              id: host
+                        {
+                            controller: 'foreman_bootdisk/disks',
+                            action: 'host',
+                            id: host
                           },
                           class: 'la'
                         )
 
       full_host_image_link = display_bootdisk_link_if_authorized(
                                _("Full host '%s' image") % host.name.split('.')[0],
-                               {
-                                   controller: 'foreman_bootdisk/disks',
-                                   action: 'full_host',
-                                   id: host
+                             {
+                                 controller: 'foreman_bootdisk/disks',
+                                 action: 'full_host',
+                                 id: host
```

Comment:
Just an advice, try to stay away from these reformatting commits. On big projects like Foreman, everyone has a preference and this usually ends up in never ending stream of hunks fighting each other. Unless you want to introduce a Rubocop rule to enforce something, in that case do it in a separate PR.

Just for the future, you can keep it not big deal.